src/encoded/static/components/cart/facet.js

```diff
@@ -0,0 +1,1418 @@
+import React from 'react';
```

Comment:
cut and paste dump to refactor out facet... Now the previous mass code deletion makes sense.